auth0/README.md

```diff
@@ -2,7 +2,37 @@
 
 ## Overview
 
-Use the Datadog-Auth0 integration to view and analyze your log events from Auth0.
+Auth0, the identity platform for development teams, provides developers and enterprises with the building blocks they need to secure their applications.
+
+
+### Integration overview
```

Comment:
```suggestion
```

I think we can skip this section to allow the following  headers to be more prominent